app/code/community/Aligent/TimedBanners/Block/Adminhtml/System/Config/Datetime.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Class Aligent_TimedBanners_Block_Adminhtml_System_Config_Datetime
+ */
+class Aligent_TimedBanners_Block_Adminhtml_System_Config_Datetime extends Mage_Adminhtml_Block_System_Config_Form_Field {
+    /**
+     * @param Varien_Data_Form_Element_Abstract $element the element to generate the html for
+     * @return string the element html
+     */
+    protected function _getElementHtml(Varien_Data_Form_Element_Abstract $element)
+    {
+        $date = new Varien_Data_Form_Element_Date();
+        $format = Mage::app()->getLocale()->getDateTimeFormat(Mage_Core_Model_Locale::FORMAT_TYPE_SHORT);
```

Comment:
Nice...  Only concern here is that we use locale functionality to get the format here, and hard code it in the helper.